.github/DISCUSSION_TEMPLATE/issue-triage.yml

```diff
@@ -0,0 +1,139 @@
+labels: ["needs confirmation"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > [!IMPORTANT]
+        > Please read through [the Discussion rules](https://github.com/ghostty-org/ghostty/discussions/6937), review [the FAQs](https://ghostty.org/docs/help#common-issues-and-solutions), and check for both existing [Discussions](https://github.com/ghostty-org/ghostty/discussions?discussions_q=) and [Issues](https://github.com/ghostty-org/ghostty/issues?q=sort%3Areactions-desc) prior to opening a new Discussion.
+  - type: markdown
+    attributes:
+      value: "# Issue Details"
+  - type: textarea
+    attributes:
+      label: Issue Description
+      description: |
+        Provide a detailed description of the issue.  Include relevant information, such as:
+        - The feature or configuration option you encounter the issue with.
+        - The expected behavior.
+        - The actual behavior (and how it deviates from the expected behavior, if it is not immediately obvious).
+        - Relevant Ghostty logs or other stacktraces.
```

Comment:
Honestly I don't trust users to decide for themselves what is "relevant". Better to just include the entire logs.